src/nvim/testdir/test_eval.in

```diff
@@ -122,7 +121,19 @@ call SetReg('/', ['abc/'])
 call SetReg('/', ["abc/\n"])
 call SetReg('=', ['"abc/"'])
 call SetReg('=', ["\"abc/\n\""])
-
+$put ='{{{1 System clipboard'
+" Save and restore system clipboard.
+" If no connection to X-Server is possible, test should succeed.
+:let _clipreg = ['+', getreg('+'), getregtype('+')]
+:let _clipopt = &cb
+:let &cb='unnamedplus'
+:1y
+:AR +
+:tabdo :windo :echo "hi"
```

Comment:
Now that I think about it, this test does nothing for us (it always succeeds, since the clipboard provider is never loaded for this test). Also, now that I *really* think about it, why *would* `:tabdo windo echo 'hi'` touch the `*` register? Makes no sense to me.